model/host/host.go

```diff
@@ -736,14 +736,8 @@ func FindHostsToTerminate() ([]Host, error) {
 				StatusKey:      evergreen.HostDecommissioned,
 			},
 			{ // unreachable
-				StatusKey: evergreen.HostUnreachable,
-				"$or": []bson.M{
-					{LastCommunicationTimeKey: bson.M{"$lt": now.Add(-unreachableCutoff)}},
-					{
-						NeedsNewAgentKey:         false,
-						LastCommunicationTimeKey: bson.M{"$gt": time.Unix(0, 0)},
-					},
-				},
+				StatusKey:                bson.M{"$in": []string{evergreen.HostUnreachable, evergreen.HostDecommissioned}},
```

Comment:
Should this just terminate hosts marked unreachable? How come we need to check the cutoff time if it's already marked unreachable?

Reply:
It actually doesn't look like we use `evergreen.HostUnreachable` anywhere. I removed it entirely and am using this status only for decommissioned hosts.